content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,321 @@
+content_type: concept
+---
+
+<!-- overview -->
+
+This checklist aims at providing a basic list of guidance with links to more
+comprehensive documentation on each topic. It does not claim to be exhaustive
+and is meant to evolve.
+
+The order of topics does not reflect an order of priority.
+
+<!-- body -->
+
+## Authentication & Authorization
+
+- [ ] `system:masters` group is not routinely used for authentication.
+- [ ] The root certificate is protected (either an offline CA, or a managed
+  online CA with effective access controls)
+- [ ] Intermediate and leaf certificates have an expiry date no more than 3
+  years in the future.
+- [ ] A process exists for periodic access review, and reviews occur no more
+  than 24 months apart
+- [ ] Kube-controller-manager is running with
+  `--use-service-account-credentials` enabled
+
+See the [Role Based Access Control Good Practices](/docs/concepts/security/rbac-good-practices/)
+for information related to authentication and authorization.
+
+<!-- New authentication certificates should be generated on new clusters that avoid -->
+<!-- the use of the `system:masters`. `system:masters` should be used as a -->
+<!-- break-glass mechanism as opposed to an admin user. -->
+
+## Network security
+
+- [ ] CNI plugins in-use supports network policies.
+- [ ] Network policies to filter how workloads and services can communicate
+  inside of the cluster are enabled.
+- [ ] Use of a service mesh to encrypt all communications inside of the cluster.
+- [ ] The Kubernetes API is not exposed publicly on Internet.
+- [ ] Access from the workloads to the cloud metadata API is filtered.
+
+A number of [Container Network Interface (CNI) plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
+plugins provide the functionality to
+restrict network resources pods may communicate with. This is most commonly done
+through [Network Policies](/docs/concepts/services-networking/network-policies/)
+which provide a namespaced resource to define rules.
+
+Not all CNI plugins provide encryption in transit. If the chosen plugin lacks this
+feature, an alternative solution could be to use a service mesh to provide that
+functionality.
+
+The `etcd` datastore of the control plane should have controls to limit access.
+Furthermore, mutual TLS (mTLS) should be used to communicate securely with it.
+The certificate authority for this should be unique to `etcd`.
+
+External Internet access to the Kubernetes API server should be restricted
+to not expose the API publicly. You can then use a bastion host to access the server.
+
+If a cloud provider is used for hosting Kubernetes, the access from pods to the cloud
+metadata API `169.254.169.254` should also be restricted or blocked if not needed
+because it may leak information.
+
+## Pod security
+
+- [ ] RBAC right to `create` workloads is only granted if necessary.
+- [ ] Appropriate pod security is enforced for all workloads.
```

Comment:
Question: Is this referring to pod security policies? Or security practices in general?

Reply:
Mmmh you are right, this one might be way too broad and not that useful, I will precise this!